plugin/al/plugin/AL_USDMayaTestPlugin/AL/usdmaya/commands/test_ProxyShapeSelect.cpp

```diff
@@ -964,34 +1153,61 @@ TEST(ProxyShapeSelect, selectSamePathTwice)
   // select a single path
   MGlobal::executeCommand("select -cl;");
   MGlobal::executeCommand("AL_usdmaya_ProxyShapeSelect -r -pp \"/root/hip1/knee1/ankle1/ltoe1\" \"AL_usdmaya_ProxyShape1\"", false, true);
-  MSelectionList sl;
-  MGlobal::getActiveSelectionList(sl);
-  EXPECT_EQ(1u, sl.length());
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1/ltoe1")));
+  if(!MayaUsdProxyShapePlugin::useVP2_NativeUSD_Rendering())
+  {
+    MSelectionList sl;
+    MGlobal::getActiveSelectionList(sl);
+    EXPECT_EQ(1u, sl.length());
```

Comment:
By virtue of the UFE selection list being a superset of the Maya selection, can't you just use the UFE selection throughout?  How can it be that the UFE selection would be inconsistent with the Maya selection?  If there's a way that this can happen, we need to plug that hole really, really quickly.